modules/core/23-commitment/types/codec_test.go

```diff
@@ -1,39 +1,41 @@
 package types_test
 
 import (
+	"fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
 
 	ibc "github.com/cosmos/ibc-go/v9/modules/core"
 	"github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types"
-	"github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
+	v2 "github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
```

Comment:
pretty sure linter will complain about the `v2` alias existing

```suggestion
	"github.com/cosmos/ibc-go/v9/modules/core/23-commitment/types/v2"
```